cmd/workerpodautoscaler/run.go

```diff
@@ -33,6 +33,11 @@ var (
 	runExample = `  workerpodautoscaler run`
 )
 
+const (
+	sqsQueueService       = "sqs"
+	beanstalkQueueService = "beanstalkd"
+)
```

Comment:
Please move these constants to https://github.com/practo/k8s-worker-pod-autoscaler/blob/master/pkg/queue/queueing_service.go

Reply:
Moved this to queuing_service.go